Leanplum-SDK/Classes/Leanplum.h

```diff
@@ -369,6 +369,13 @@ typedef enum {
  */
 + (void)onceVariablesChangedAndNoDownloadsPending:(LeanplumVariablesChangedBlock)block;
 
+/**
+ * Clear all local variables and messages.
+ * This should be typically called when a user logs out of your app. Variables will
+ * start sending default values until the server sends updated content.
+ */
++ (void)clearUserContent;
```

Comment:
This method is being exposed publicly here for customers to use. The use case is:
1. User A logs in to DeviceX
2. User A receives messages / variables
3. User A logs out
4. User B logs in to DeviceX
5. The start/FCU call to the server times out
6. User B gets User A's variants.

To prevent this, we enable customers to clear the cached content with this method